tests/components/recorder/test_util.py

```diff
@@ -60,3 +62,31 @@ def to_native(validate_entity_id=True):
         util.execute((mck1,), to_native=True)
 
     assert e_mock.call_count == 2
+
+
+def test_validate_or_move_away_sqlite_database(hass, tmpdir, caplog):
+    """Ensure a malformed sqlite database is moved away."""
+
+    test_dir = tmpdir.mkdir("test_validate_or_move_away_sqlite_database")
+    test_db_file = f"{test_dir}/broken.db"
+    dburl = f"{SQLITE_URL_PREFIX}{test_db_file}"
+
+    util.validate_sqlite_database(test_db_file) is True
+
+    assert os.path.exists(test_db_file) is True
+    assert util.validate_or_move_away_sqlite_database(dburl) is True
```

Comment:
Codacy found an issue: [Use of assert detected. The enclosed code will be removed when compiling to optimised byte code.](https://app.codacy.com/manual/foreign-sub/home-assistant/pullRequest?prid=5872960)